lamindb/models/collection.py

```diff
@@ -342,10 +345,21 @@ def append(self, artifact: Artifact, run: Run | None = None) -> Collection:
             run=run,
         )
 
-    def open(self, is_run_input: bool | None = None) -> PyArrowDataset:
+    def open(
+        self,
+        df_engine: Literal["pyarrow", "polars"] = "pyarrow",
+        is_run_input: bool | None = None,
+        **kwargs,
+    ) -> PyArrowDataset | Iterator[PolarsLazyFrame]:
         """Return a cloud-backed pyarrow Dataset.
 
-        Works for `pyarrow` compatible formats.
+        Works for `pyarrow` and `polars` compatible formats.
```

Comment:
Can you spell this out? Does this mean "a collection of parquet files"?

Reply:
I would prefer to keep it like this because there are a lot of formats, and they are a bit different between `pyarrow` and `polars`. It is not supported, an error is thrown describing the required suffixes.

Reply:
I'm just asking to give some examples for what "compatible formats" are, e.g. mention a collection of parquet files because this is what people know.

Reply:
done